test/package.json

```diff
@@ -25,28 +25,28 @@
     "fmt": "tslint -p . --fix && prettier 'src/**/*.{ts, json}' --write"
   },
   "devDependencies": {
-    "@types/jest": "^23.3.1",
     "@types/chai": "^4.2.5",
     "@types/chai-as-promised": "^7.1.2",
+    "@types/jest": "^23.3.1",
     "@types/lodash": "^4.14.149",
     "@types/mkdirp": "^0.5.2",
     "@types/mocha": "^5.2.7",
     "@types/ncp": "^2.0.3",
     "@types/node": "^12.12.12",
-    "@types/seedrandom": "^2.4.28",
-    "@types/snappy": "^6.0.0",
     "@types/node-fetch": "2.1.2",
     "@types/request-promise": "^4.1.42",
-    "mocha": "^6.2.2",
+    "@types/seedrandom": "^2.4.28",
+    "@types/snappy": "^6.0.0",
     "jest": "^23.5.0",
+    "mocha": "^6.2.2",
     "nodemon": "^1.17.5",
     "prettier": "^1.19.1",
-    "ts-node": "^8.5.2",
     "ts-jest": "^23.1.3",
+    "ts-node": "^8.5.2",
     "tslint": "^5.20.1",
     "tslint-config-prettier": "^1.18.0",
-    "typedoc": "^0.14.2",
     "tslint-no-unused-expression-chai": "^0.1.4",
+    "typedoc": "^0.16.11",
     "typescript": "^3.7.2"
```

Comment:
Please remove unrelated changes. Or you can create a commit for this.

Reply:
Sorry for making mistakes often.  This was not intended. I'll remove it.